brats/transformations.py

```diff
@@ -73,3 +78,77 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
```

Comment:
In the `torch` documentation ([link](https://pytorch.org/docs/stable/nn.html#torch.nn.Conv3d)), it stands that the Conv3d accepts the input (N, C, D, **H, W**) and your transformation changes it to (C, D, **W, H**). Shouldn't these two dimensions match?

Reply:
It does match in code, this is a mistake in docstring.